hiedb.cabal

```diff
@@ -15,9 +15,7 @@ extra-source-files:
   README.md
   test/data/*.hs
   test/data/Sub/*.hs
-tested-with:         GHC ==8.8.1 || ==8.8.2  || ==8.8.3  || ==8.8.4
-                     || ==8.10.1 || ==8.10.2 || ==8.10.3 || ==8.10.4
-                     || ==9.0.1
+tested-with:         GHC ==8.8.4 || ==8.10.7 || ==9.0.2 || ==9.2.2
```

Comment:
What do you think of this change? Does it make sense to only keep the latest patch version of each major ghc release?